diff --git a/kernel/sched/core.c b/kernel/sched/core.c
index 0c19b0d9d65f..d765fbaa6a7f 100644
--- a/kernel/sched/core.c
+++ b/kernel/sched/core.c
@@ -9463,8 +9463,7 @@ static int tg_set_cfs_bandwidth(struct task_group *tg, u64 period, u64 quota,
 	if (quota != RUNTIME_INF && quota > max_cfs_runtime)
 		return -EINVAL;
 
-	if (quota != RUNTIME_INF && (burst > quota ||
-				     burst + quota > max_cfs_runtime))
+	if (quota != RUNTIME_INF && burst + quota > max_cfs_runtime)
 		return -EINVAL;
 
 	/*
