Index: linux-trixie-6.19/kernel/sched/core.c
===================================================================
--- linux-trixie-6.19.orig/kernel/sched/core.c	2026-05-07 11:16:40.583671382 +0000
+++ linux-trixie-6.19/kernel/sched/core.c	2026-05-07 11:17:04.808970660 +0000
@@ -9758,8 +9758,7 @@
 	if (quota_us != RUNTIME_INF && quota_us > max_bw_runtime_us)
 		return -EINVAL;
 
-	if (quota_us != RUNTIME_INF && (burst_us > quota_us ||
-					burst_us + quota_us > max_bw_runtime_us))
+	if (quota_us != RUNTIME_INF && (burst_us + quota_us > max_bw_runtime_us))
 		return -EINVAL;
 
 #ifdef CONFIG_CFS_BANDWIDTH
